source/Octo.Tests/Octo.Tests.csproj

```diff
@@ -30,7 +30,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Nancy" Version="2.0.0-clinteastwood" />
-    <PackageReference Include="Octopus.Client" Version="8.8.8" />
+    <PackageReference Include="Octopus.Client" Version="11.3.3415" />
```

Comment:
This version will be bumped again once the new client version is merged and built. The big change from v8 to v11 brought forward some of the additional changes in this commit